cmd/statusd/library.go

```diff
@@ -21,7 +21,7 @@ func GenerateConfig(datadir *C.char, networkID C.int, devMode C.int) *C.char {
 		return makeJSONResponse(err)
 	}
 
-	outBytes, err := json.Marshal(&config)
+	outBytes, err := json.Marshal(config)
```

Comment:
Just notifying that generally we must not make unrelated changes because they may impede merging sometimes. This change won't interfere with anything else so it's fine.